TODO

```diff
@@ -0,0 +1 @@
+When the OPENTURNS_MAXSCALAR_IS_INF compilation option is removed, some checks marked TODO JM need to be removed as well.
```

Comment:
maybe we could also rename MaxScalar to Infinity and ActualMaxScalar to MaxScalar when that option disappears

Reply:
I agree, added to TODO.